lib/db/schema/patch-015-016.sql

```diff
@@ -0,0 +1,46 @@
+-- Add queries for metrics reporting
+
+CREATE PROCEDURE `countAccountsCreatedBefore_1` (
+    IN createdBefore BIGINT UNSIGNED
+)
+BEGIN
+    SELECT COUNT(*) AS count FROM accounts
+    WHERE createdAt < createdBefore;
+END;
+
+CREATE PROCEDURE `countVerifiedAccountsCreatedBefore_1` (
+    IN createdBefore BIGINT UNSIGNED
+)
+BEGIN
+    SELECT COUNT(*) AS count FROM accounts
+    WHERE createdAt < createdBefore
+    AND emailVerified = true;
+END;
+
+CREATE PROCEDURE `countMultiDeviceAccounts_1` ()
+BEGIN
+    SELECT COUNT(*) AS count FROM (
+        SELECT uid FROM sessionTokens
+        GROUP BY uid
+        HAVING COUNT(tokenId) > 1
+    ) AS s;
+END;
+
+CREATE PROCEDURE `countThreePlusDeviceAccounts_1` ()
+BEGIN
+    SELECT COUNT(*) AS count FROM (
+        SELECT uid FROM sessionTokens
+        GROUP BY uid
+        HAVING COUNT(tokenId) > 2
+    ) AS s;
+END;
+
+CREATE PROCEDURE `countMobileDeviceAccounts_1` ()
+BEGIN
+    SELECT COUNT(DISTINCT uid) AS count
+    FROM sessionTokens
+    WHERE uaDeviceType = 'mobile';
```

Comment:
I worry about the performance of `COUNT DISTINCT`, although I don't yet have anything concrete to back that up.  Let's see about trying these out in e.g. stage to get a feel for how they perform.